docs/guides/mocha.md

```diff
@@ -24,9 +24,3 @@ describe('<Foo />', () => {
 });
 
 ```
-
-
-## Example Projects
```

Comment:
do we no longer have example projects? or is it not needed because of removing react-compat

Reply:
IMO these projects were never maintained and have not been updated. if we want to add some to this repo, i think that might help keep them updated... but these are no longer good references, so i've removed them.

Reply:
Sounds good